neqo-client/src/main.rs

```diff
@@ -75,8 +75,21 @@ pub struct Args {
     #[structopt(short = "h", long, number_of_values = 2)]
     header: Vec<String>,
 
-    #[structopt(name = "max-table-size", short = "t", long, default_value = "128")]
-    max_table_size: u64,
+    #[structopt(
+        name = "max-table-size_encoder",
```

Comment:
I think that the mix of hyphens and underscores will be annoying.  I know that the German keyboard layout finds both equally annoying, but the typical US keyboard doesn't have a modifier key for hyphens.  I recommend just using `encoder-table-size`.